projects/angular-calendar/schematics/utils/config.ts

```diff
@@ -0,0 +1,53 @@
+import { Tree, SchematicsException } from '@angular-devkit/schematics';
+
+const CONFIG_PATH = 'angular.json';
+
+export function addStyle(host: Tree, stylePath: string): void {
+  const config = readConfig(host);
+  const appConfig = getAngularAppConfig(config);
+
+  if (appConfig) {
+    appConfig.architect.build.options.styles.push({
```

Comment:
Let's add the styles to the start of the array instead of the end (`appConfig.architect.build.options.styles.unshift`), so that it will be easier to override them in your app styles below.

Reply:
Also let's just add `stylePath` instead of `{ input: stylePath }`